File: Likelihood/Python/Likelihood.py
import numpy as np
import matplotlib.pyplot as plt

n = 400
y = 72
theta = np.arange(0.01,1,0.01)

def likelihood(n,y,theta):
    return(theta**y*(1-theta)**(n-y))

plt.plot(theta,likelihood(n, y, theta))
plt.axvline(x=y/n)
plt.ylabel("Likelihood")
plt.xlabel("Theta")
plt.show()
plt.clf()

def loglike(n,y,theta):
    return(np.log(theta)*y+np.log(1-theta)*(n-y))

plt.plot(theta,loglike(n,y,theta)) 
plt.axvline(x=y/n)
plt.ylabel("Log-likelihood")
plt.xlabel("Theta")
plt.show()
plt.clf()

File: Likelihood/R/Likelihood.R
likelihood <- function(n,y,theta){return(theta^y*(1-theta)^(n-y))}
loglike <- function(n,y,theta){return(log(theta)*y+log(1-theta)*(n-y))}
n <- 400
y <- 72
theta <- seq(0.01, 0.99, 0.01)
plot(theta, likelihood(n,y,theta),"l")
abline(v=y/n)

plot(theta, loglike(n,y,theta),"l")
abline(v=y/n)